docsrc/Managing_resources.rst

```diff
@@ -17,80 +17,118 @@ To get started, follow the steps below:
 
 **1. Import modules**
 
-	To initialize a ``ResourceManager`` object, import the modules shown below.
+    To initialize a ``ResourceManager`` object, import the modules shown below.
 
 .. _required_resourcemanager_imports:
 
-	::
+    ::
 
-		from firebolt.service.manager import ResourceManager
-		from firebolt.common import Settings
+        from firebolt.client.auth import UsernamePassword
+        from firebolt.service.manager import ResourceManager
 
+**2. Initialize a ResourceManager object**
+
+    A ResourceManager object contains the user credentials and other information needed to
+    manage Firebolt databases and engines.
+
+    ResourceManager uses the following parameters:
+
+    +---------------------+-----------------------------------------------------------------------------------------------------------------------------+
+    | ``auth``            |  Auth object, containing your credentials. See :ref:`Auth <firebolt.client:auth>` for more details.                         |
+    +---------------------+-----------------------------------------------------------------------------------------------------------------------------+
+    | ``api_endpoint``    |  The API hostname for logging in. Defaults to ``api.app.firebolt.io`` if not included.                                      |
+    +---------------------+-----------------------------------------------------------------------------------------------------------------------------+
+    | ``account_name``    |  The name of the account you're using to connect to Firebolt. Must be specified in order to authenticate.                   |
+    +---------------------+-----------------------------------------------------------------------------------------------------------------------------+
+
+
+    ::
+
+        rm = ResourceManager(
+            auth=UsernamePassword("your_username", "your_password"),
+            account_name="your_acc_name",
+        )
+
+
+Setting up a ResourceManager object (legacy)
+================================================
+
+**1. Import modules**
+
+    To initialize a ``ResourceManager`` object, import the modules shown below.
+
+.. _required_resourcemanager_imports:
+
+    ::
+
+        from firebolt.client.auth import UsernamePassword
+        from firebolt.service.manager import ResourceManager
 
 **2. Initialize a Settings object**
 
-	A Settings object contains the user credentials and other information needed to
```

Comment:
I belive we shouldn't mention `Settings` for a 1.x version since it's deprecated and shouldn't be used

Reply:
This is a PR for 0.x branch, 1.x does not have any mention of settings - https://github.com/firebolt-db/firebolt-python-sdk/blob/main/docsrc/Managing_resources.rst
This RM setup section comes second after the one with direct parameters and is labeled `(legacy)`. Some users of 0.x might still be using Settings so we should keep this until FB1.0 is deprecated altogether.